utils/maven-plugin/src/main/java/io/apicurio/registry/maven/AbstractRegistryMojo.java

```diff
@@ -64,11 +66,13 @@
     String password;
 
     private static RegistryClient client;
+    private static ApicurioHttpClient httpClient;
 
     protected RegistryClient getClient() {
         if (client == null) {
             if (authServerUrl != null && clientId != null && clientSecret != null) {
-                Auth auth = new OidcAuth(authServerUrl, clientId, clientSecret, Optional.empty());
+                httpClient = ApicurioHttpClientFactory.create(authServerUrl, new AuthErrorHandler());
```

Comment:
Who closes this one?

Reply:
This same class, but I forgot to add it at the end of the execute method.

Reply:
Fixed.